lib/ntpUtil.js

```diff
@@ -29,6 +57,62 @@ const generatePublicKeyAndID = (privateKeyFile) => {
   }
 }
 
+const isSupportedSchemaVersion = (version) => {
+  return version === 1 || version === 2;
```

Comment:
If this gets larger (ex: we have version 3, 4, 5...), we can always change to something like `return [1,2].includes(version)`

Reply:
Oops, this code was not my final code. Updated.
At first, I tried to introduce schema version 2. but I think it's fine to use 1 for new spec because this new change is just adding new properties and not changing current properties. WDYT?

Reply:
SGTM! 😄